frontend/src/features/contacts/contact/detail/ContactInfoSubForm.tsx

```diff
@@ -0,0 +1,75 @@
+import { ContactMethodTypes } from 'constants/contactMethodType';
+import { ContactInfoField } from 'features/contacts/interfaces';
+import { Dictionary } from 'interfaces/Dictionary';
+import { IContactEntity } from 'interfaces/IContact';
+import * as React from 'react';
+import { Col } from 'react-bootstrap';
+import { phoneFormatter } from 'utils/formUtils';
+
+import * as Styled from '../../styles';
+import { getContactInfo } from './utils';
+
+interface IContactInfoSubFormProps {
+  contactEntity: IContactEntity;
+}
+
+const phoneTypes: Dictionary<string> = {};
+phoneTypes[ContactMethodTypes.WorkMobile] = 'Mobile';
```

Comment:
@FuriousLlama note that this list of contact methods was different for the organization and for the person. This seems strange since the list of contact methods provided on the person/organization edit screens is the same.